### ISSUE TITLE
RDP wrap Win10 10.0.18362.657, can't support 2 users remote access 

### ISSUE THREAD

**Issue by peter7802** [stascorp/rdpwrap#1013]
I download the latest file and ini file, and run conf.exe show all green. but it still can't support two people login at the same time.

![image](https://user-images.githubusercontent.com/57980536/75529364-6d8b0e00-5a4e-11ea-8476-e2098d5a211c.png)

**Comment by Veerapoom** [comment]
You need to uncheck Single session per user to allow mutiple session.

**Comment by vfernandes0408** [comment]
I`m having the same issue. I uncked single session but the problem persist.

**Comment by maestro2369** [comment]
I'm having the issue too. Hope we'll find a solution quickly ! :)

**Comment by anacierdem** [comment]
![image](https://user-images.githubusercontent.com/6522751/75632247-39108100-5c0b-11ea-9208-392bc7ba24fe.png)
Same here. Difference is mine says [not supported]

**Comment by TwoQuantumBits** [comment]
I ran into this problem as well until I found out on #859 that there is a new .ini file to support this version of Windows. Here's what I did:

1. Stop Remote Desktop Services (in services.msc) - This is mandatory or you'll have to restart your computer for it to (probably) work.
2. In the first post of #859, download "rdpwrap.ini from 16 Feb 2020".
3. Replace the original rdpwrap.ini in Program Files/RDP Wrapper with the downloaded one.
4. Restart Remote Desktop Services.

**Comment by breezeliusong** [comment]
I have the same issue, it still doesn't work after I followed your steps. I can only login only one user one time. Any one has any idea?

**Comment by TwoQuantumBits** [comment]
> I have the same issue, it still doesn't work after I followed your steps. I can only login only one user one time. Any one has any idea?

How about you do the same thing, but with the one in #988?

**Comment by breezeliusong** [comment]
unfortunately, it still doesn't work, I have the same condition as the @peter7802 in this issue. Thanks @trandanggiabao0203 . Maybe I should give it up.

**Comment by isedaniel** [comment]
> 
> 
> > I have the same issue, it still doesn't work after I followed your steps. I can only login only one user one time. Any one has any idea?
> 
> How about you do the same thing, but with the one in #988?

This one worked perfectly for me

**Comment by leebinx** [comment]
#988 fixed my problem, thanks!
if having a problem after fixing, try restarting the computer.

**Comment by Lotus-Blue** [comment]
> #988 fixed my problem, thanks!
> if having a problem after fixing, try restarting the computer.

thank you

**Comment by breezeliusong** [comment]
Amazing, I tried to uninstall and reinstall the RDP, then replaced the .ini file in #[988 ](https://github.com/stascorp/rdpwrap/issues/988). then I changed the Authentication Mode to Default RDP authentication, it worked, I could sign in with both users at the same time. Then I changed back to GUI Authentication Only, it still  worked. But I don't care what it goes, it works for me now. Thanks everybody, appreciate all your helps.
